### ISSUE TITLE
dismiss icon in snackbar queue is not correctly rendered

### ISSUE THREAD

**Issue by dardinier** [jamesmfriedman/rmwc#560]
- **What RMWC Version are you using [major.minor.patch]:**

5.7.2

- **Name your build system [Webpack, Rollup...]:**

webpack 4

- **Describe the bug with as much detail as possible:**

I used snackbar queue to push notifications this way :

```jsx
import { createSnackbarQueue, SnackbarQueue } from "@rmwc/snackbar";

const queue = createSnackbarQueue();

const App = () => (
  <React.Fragment>
    <Button
      raised
      onClick={() => queue.notify({ title: "Hi there", dismissIcon: true })}
    >
      Notify
    </Button>
    <SnackbarQueue messages={queue.messages} />
  </React.Fragment>
);
```

The dismiss icon hover state is not correctly rendered (not centered on the icon) :

![Screenshot_20200303_143005](https://user-images.githubusercontent.com/24563898/75780409-9287c580-5d5b-11ea-8205-afb38cb67c46.png)

(Once clicked, the ripple is centered) :
![Screenshot_20200303_143023](https://user-images.githubusercontent.com/24563898/75780447-a7645900-5d5b-11ea-8fbb-029098d03c99.png)

Example here : https://codesandbox.io/s/rmwc-sandbox-r6wgl

- **What was supposed to happen:**

The hover state should be centered on the icon.

It should look like this (example from MDC web demo page) :

![Screenshot_20200303_143429](https://user-images.githubusercontent.com/24563898/75780886-79334900-5d5c-11ea-9542-551cbabf3186.png)

**Comment by jamesmfriedman** [comment]
Definitely see the issue you're talking about. I have to imagine it's a pretty low level issue as to when the ripple actually gets measured. It would seem that it is getting messed up by the css transform.

**Comment by jamesmfriedman** [comment]
Fixed in 6.0.0
